bescort.lic

```diff
@@ -132,6 +132,10 @@ class Bescort
         { name: 'mode', options: %w(acen fang ratha), description: 'Where do you need to get to?' }
       ],
       [
+        { name: 'sandbarge', regex: /sandbarge/i, description: 'The sand barge between Hvaral, Oasis, and Muspari' },
+        { name: 'mode', options: %w(mus_hra mus_oas oas_mus oas_hra hra_mus hra_oas), description: 'Where are you coming from?_Where do you need to get' }
```

Comment:
I originally was going that route and reverted.  I was using examples in the code from the file and wanted to keep it consistent.